openlibrary/macros/NotInLibrary.html

```diff
@@ -1,6 +1,5 @@
-$def with(work_key)
+$def with(key)
```

Comment:
```suggestion
$def with()
```
Let's remove this parameter as it is now unused.